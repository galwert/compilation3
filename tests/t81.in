void main() // Piazza @89
{
    bool hi1 = true if(true) else  1b==2 ;
    bool hi2 = true if(true) else  (1b==2) ;
}