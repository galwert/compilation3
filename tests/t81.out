---end scope---
hi1 BOOL 0
hi2 BOOL 1
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
